aztec/src/main/res/drawable/format_bar_button_media_selector.xml

```diff
@@ -6,6 +6,7 @@
     <item android:drawable="@drawable/format_bar_button_media_disabled" android:state_enabled="false" />
     <item android:drawable="@drawable/format_bar_button_media_highlighted" android:state_pressed="true" />
     <item android:drawable="@drawable/format_bar_button_media_highlighted" android:state_focused="true" />
+    <item android:drawable="@drawable/format_bar_button_media_highlighted" android:state_checked="true" />
```

Comment:
Can we move the `state_checked` entry between the `state_enabled` and `state_pressed` entries to mimic the other selector files (e.g. [`format_bar_button_bold_selector`](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/aztec/src/main/res/drawable/format_bar_button_bold_selector.xml))?  That way we can avoid potential bugs from differing behavior.